sympy/functions/elementary/integers.py

```diff
@@ -86,25 +86,32 @@ class floor(RoundFunction):
     value not greater than its argument. However this implementation
     generalizes floor to complex numbers.
 
-    More information can be found in "Concrete mathematics" by Graham,
-    pp. 87 or visit http://mathworld.wolfram.com/FloorFunction.html.
-
-        >>> from sympy import floor, E, I, Float, Rational
-        >>> floor(17)
-        17
-        >>> floor(Rational(23, 10))
-        2
-        >>> floor(2*E)
-        5
-        >>> floor(-Float(0.567))
-        -1
-        >>> floor(-I/2)
-        -I
+    Examples
+    ========
+
+    >>> from sympy import floor, E, I, Float, Rational
+    >>> floor(17)
+    17
+    >>> floor(Rational(23, 10))
+    2
+    >>> floor(2*E)
+    5
+    >>> floor(-Float(0.567))
+    -1
+    >>> floor(-I/2)
+    -I
 
     See Also
     ========
 
-    ceiling
+    sympy.functions.elementary.integers.ceiling
```

Comment:
It's a bit unfortunate because on one hand, the module structure is an implementation detail that should not bother users and might change from one release of SymPy to the next, but on the other hand, the autogenerated HTML derives directly from the module structure.
If the above is turned into a link, then it should quote the module in which `ceiling` is documented, which might be the one that `ceiling` is defined in or one of its parent packages.